crates/turborepo-ffi/src/scm/mod.rs

```diff
@@ -0,0 +1 @@
+pub(crate) mod git;
```

Comment:
This should be its own crate so we can use SCM in Rust without requiring protobuf to be setup

Reply:
Should we have crates for all the dependencies we're porting? That seems a little too granular.

Reply:
I'm in favor of being too granular with our crates for the time being. If the overhead for adding crates is too much then we can add a `turborepo-util`, but I'm concerned that will just become a dumping ground.